dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1218,7 +1218,7 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 						/>
 					</Island>
 
-					{isWeb && !isPaidContent && article.isCommentable && (
+					{!isPaidContent && showComments && (
```

Comment:
Should `!isPaidContent` be part of the `showComments` logic?

IIUC, we might show the comment count, but not the comment section with the current logic, which is probably innacurate

Reply:
I think there’s a similar thing happening in [`LiveLayout`](https://github.com/guardian/dotcom-rendering/pull/9634#discussion_r1405962445)